src/ILCompiler.Compiler/src/ILCompiler.Compiler.csproj

```diff
@@ -38,6 +38,7 @@
     <Compile Include="Compiler\CompilerTypeSystemContext.cs" />
     <Compile Include="Compiler\DelegateInfo.cs" />
     <Compile Include="Compiler\DependencyAnalysis\ArrayOfEmbeddedDataNode.cs" />
+    <Compile Include="Compiler\DependencyAnalysis\EETypeFlags.cs" />
```

Comment:
Nice; didn't realise we had managed forms of these enums already. I'll do something here to remove the duplication.